### PR TITLE
pangolin stub, nf-test, snap

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

File: modules/nf-core/pangolin/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.report.collect { file(it[1]).getName() } +
+                                process.out.versions).match() }
```
Comment (CarsonJM) [inline modules/nf-core/pangolin/tests/main.nf.test:52]: Empty hash in stube tests shouldn't cause file to fail nf-tests anymore! https://github.com/nf-core/tools/pull/2571

**Review by CarsonJM — APPROVED** [review]
Any idea what's going on with the conda tests?

**Comment by sateeshperi** [comment]
not sure whats up with conda...have to check the bioconda recipe

File: modules/nf-core/pangolin/tests/main.nf.test (hunk 1 above)
Comment (sateeshperi) [reply modules/nf-core/pangolin/tests/main.nf.test:52]: will update this

**Comment by SPPearce** [comment]
csv output from docker:
```
taxon,lineage,conflict,ambiguity_score,scorpio_call,scorpio_support,scorpio_conflict,scorpio_notes,version,pangolin_version,scorpio_version,constellation_version,is_designated,qc_status,qc_notes,note
"MT192765.1 Severe acute respiratory syndrome coronavirus 2 isolate SARS-CoV-2/human/USA/PC00101P/2020, complete genome",B.1,0.0,,,,,,PANGO-v1.17,4.2,0.3.17,v0.1.10,True,pass,Ambiguous_content:0.02,Assigned from designation hash.
```
csv output from conda:
```
taxon,lineage,conflict,ambiguity_score,scorpio_call,scorpio_support,scorpio_conflict,scorpio_notes,version,pangolin_version,scorpio_version,constellation_version,is_designated,qc_status,qc_notes,note
"MT192765.1 Severe acute respiratory syndrome coronavirus 2 isolate SARS-CoV-2/human/USA/PC00101P/2020, complete genome",B.1,0.0,,,,,,PANGO-v1.28,4.2,0.3.19,v0.1.12,True,pass,Ambiguous_content:0.02,Assigned from designation hash.
```

This suggests there is a different database being used.
Maybe seqera containers will fix this?

**Comment by sateeshperi** [comment]
trying with seqera containers in #6740 

can close this if that works

**Pull request closed without merging**